### DISCUSSION TITLE
Internet access when Security Goups for pods are attached to pods in public subnet

### DISCUSSION THREAD

**Discussion by krzwiatrzyk-lgd** [aws/amazon-vpc-cni-k8s#2211] [Q&A] [answered]
Hello,
I have two pods deployed that has a security group policy attached:
- one on the worker node in public subnet
- one on the worker node in private subnet

And public one is cut off from the internet. It is by default?

**Comment by jdn5126** [comment, marked as answer]
The Security Group Policy applies to all traffic egressing and ingressing the pod, regardless of the subnet that the worker node is in. Please file an issue at https://github.com/aws/amazon-vpc-cni-k8s/issues for questions like this in the future.
